### PR TITLE
Enhancement/skale 3399 handle script error

### PR DESCRIPTION
**Pull request by badrogger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,17 +9,17 @@ SKALE Node CLI, part of the SKALE suite of validator tools, is the command line
...
-    2.2 [User](#user-commands)  
-    2.3 [Node](#node-commands)  
-    2.4 [Wallet](#wallet-commands)  
-    2.5 [sChains](#schain-commands)  
-    2.6 [Containers](#containers-commands)  
-    2.7 [SGX](#sgx-commands)  
-    2.8 [SSL](#ssl-commands)  
-    2.9 [Logs](#logs-commands)  
-3.  [Development](#development)  
+2.  [CLI usage](#cli-usage)
```
Comment (dmytrotkk) [inline README.md]: return spaces plz

File: cli/node.py [hunk 2]
```diff
@@ -149,8 +149,11 @@ def signature(validator_id):
 @node.command('backup', help="Generate backup file to restore SKALE node on another machine")
 @click.argument('backup_folder_path')
 @click.argument('env_file')
-def backup_node(backup_folder_path, env_file):
-    backup(backup_folder_path, env_file)
+@click.option('--no-database', is_flag=True,
+              help="Skaip mysql backup")
```
Comment (dmytrotkk) [inline cli/node.py]: Typo here

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: cli/node.py (hunk 2 above)
Comment (badrogger) [reply cli/node.py]: addressed

File: README.md (hunk 1 above)
Comment (badrogger) [reply README.md]: addressed

**Review by dmytrotkk — APPROVED** [review]

**Pull request merged**